packages/instantsearch-react/examples/history.js

```diff
@@ -2,7 +2,17 @@ import {useRouterHistory} from 'react-router';
 import {createHistory} from 'history';
 import qs from 'qs';
 
+function alphabeticalSort(a, b) {
```

Comment:
Can [localeCompare](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) do the trick?